sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/Plan.java

```diff
@@ -21,6 +22,8 @@
  * list of one or more error messages to be shown to the user,
  */
 public interface Plan extends ParentElement<Phase> {
+    void setErrors(List<String> errors);
```

Comment:
We actually already have an appropriate constructor in `DefaultPlan` for this.  Updating and removing the setter.  Thanks for pushing on this @nickbp.